src/util/BaseLoginRouter.js

```diff
@@ -206,6 +207,13 @@ function (Okta, BrowserFeatures, Settings,
           .done();
       }
 
+      if (this.settings.get('colors.brand') && !ColorsUtil.isLoaded()) {
```

Comment:
probably could move `ColorsUtil.isLoaded` check to `ColorsUtil.addStyle` where is smart enough to add style or not base on whether it's already loaded.

Reply:
The reason I put it there is to be consistent with the similar pattern we use for i18n Bundle (https://github.com/okta/okta-signin-widget/blob/646bfaf7f467cdfea5cee3828a3fe2f19721aa0c/src/util/BaseLoginRouter.js#L199). If we are ok breaking this pattern, I can move the check to the `ColorsUtil.addStyle` function.